### DISCUSSION TITLE
Forth advice meta-question

### DISCUSSION THREAD

**Discussion by IanWitham** [jkotlinski/durexforth#530] [Q&A] [answered]
Even though I'm still using durexForth exclusively, some of the questions I have feel very "forth noob". As in, these are kind of general questions about programming in forth.
Even things like "this word definition is ugly, I'm sure there's a nicer way. Please advise!"
Is this an appropriate forum for that sort of question, or would people prefer not to see that sort of thing here?

**Comment by jkotlinski** [comment, marked as answer]
As far as I am concerned, general Forth questions are totally welcome!

ons 1 mars 2023 kl. 22:40 skrev amuletofyendor ***@***.***>:
